SPECS/nvidia-container-runtime/nvidia-container-runtime.spec

```diff
@@ -0,0 +1,62 @@
+Summary:      NVIDIA container runtime
+Name:         nvidia-container-runtime
+Version:      3.4.2
+Release:      2%{?dist}
+License:      ASL 2.0
+URL:          https://github.com/NVIDIA/nvidia-container-runtime
+Vendor:       Microsoft Corporation
+Distribution: Mariner
+#Source0:     https://github.com/NVIDIA/${name}/archive/v%{version}.tar.gz
```

Comment:
Typo: please replace `$` with `%%`. Same for the single `%`.

Why double a `%`? It prevents the spec-parsing tools from complaining that there's a macro in a comment. Not an important thing, but it generates noise.

Reply:
Fixed typo. I do not see any instances of "%%" in that file what line are you referencing there?